errbot/rendering/__init__.py

```diff
@@ -9,6 +9,12 @@
 # Here are few helpers to simplify the conversion from markdown to various
 # backend formats.
 
+_md_escape_re = re.compile('|'.join(re.escape(c) for c in Markdown.ESCAPED_CHARS))
+
+
+def _md_escape_trans(match):
```

Comment:
@gbin should this "helper" function be inside the `md_escape` function? I do not suppose it will be used anywhere else so it makes sense to encapsulate it in `md_escape`.